src/main/java/com/puppycrawl/tools/checkstyle/checks/FinalParametersCheck.java

```diff
@@ -48,6 +48,13 @@
  * Default value is {@code false}.
  * </li>
  * <li>
+ * Property {@code ignoreUnnamedParameters} -
```

Comment:
Not really for this line, but please add a new example that shows how this property works, with a brief explanation about why someone may want to use it.

Reply:
should we always do this?  I think we introduced new properties in other PRs without adding new examples for it

Reply:
> should we always do this?

Maybe not always, but probably most of the time.

Reply:
@nrmancuso

Reply:
@nrmancuso done